commands/who.js

```diff
@@ -153,6 +153,22 @@ function getComponentsForTarget(member, perm, targetMember, targetPerm, invite)
 				.setStyle(ButtonStyle.Danger);
 			modRow.addComponents(setptt);
 		}
+
+		let guestRole = targetMember.roles.cache.find(r => r.name === global.config.guestRole);
```

Comment:
Thinking about the usefulness here, add/remove guest might be best as the first in the row.

Reply:
Fixed in 1c6c85a